### ISSUE TITLE
Not shuffling the training set?

### ISSUE THREAD

**Issue by stevenygd** [NVIDIA/tacotron2#130]
https://github.com/NVIDIA/tacotron2/blob/825ffa47d18994510882f0fa4f2708f4359d5377/train.py#L62

Is there a reason why we don't shuffle the training set here? Thanks!

**Comment by rafaelvalle** [comment]
We shuffle it once: https://github.com/NVIDIA/tacotron2/blob/master/data_utils.py#L28
It's probably a good idea to shuffle it on every epoch!

**Comment by stevenygd** [comment]
I thought shuffling it on every epoch is good for batch norm. (From the original paper)

**Comment by rafaelvalle** [comment]
Exactly! Hence it's probably a good idea to shuffle on every epoch. We'll update the repo. Thanks for noticing it.
